stable/insights-agent/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 0.13.0
-appVersion: 0.6.0
+version: 0.14.0
+appVersion: 0.7.1
```

Comment:
We haven't been 100% consistent on this, but I think this will go out with release 0.8.0